app/views/pages/get-started/contact-us-tutorial/defining-the-email.liquid

```diff
@@ -0,0 +1,226 @@
+---
```

Comment:
Tested this in your instance and this page does not appear in the navigation for some reason.

Reply:
Good catch! This part was not included in views/partials/shared/nav/get-started.liquid, I fixed it.